kyuubi-main/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -96,9 +96,13 @@ class KyuubiSessionImpl(
         info(s"Launching SQL engine: $builder")
         var sh = getServerHost
         val started = System.currentTimeMillis()
+        var exitValue: Option[Int] = None
```

Comment:
unnecessary definition？

Reply:
necessary.  there is no need to execute `process.waitFor`, when the process exits normally, eg: cluster mode and 'spark.yarn.submit.waitAppCompletion=false'

Reply:
If the process exits already, it won't wait.